### DISCUSSION TITLE
希望能提供一个方便查看 css变量 的页面

### DISCUSSION THREAD

**Discussion by JF98** [element-plus/element-plus#11516] [General]
Feature type: **`Other`**

## Details

- Enhanced component name: `N/A`

### arco 的页面参考
https://arco.design/vue/docs/token

### Additional comments

_(empty)_

**Comment by 11003** [comment]
跪求！！调暗黑模式太需要了

**Comment by akbar0903** [comment]
我也觉得这个太有必要了，希望官方出这样的页面。
